src/visualizers/visualizers.ts

```diff
@@ -1,5 +1,5 @@
 const vizFiles = require.context('.', false, /\.ts$/)
-const vizMODULES: { [key: string]: Function } = {};
+const vizMODULES: { [key: string]: __WebpackModuleApi.RequireContext } = {};
```

Comment:
I don't know what's going on here. Please explain, @gwhitney.

Reply:
I will comment this in the upcoming commit. Again, thanks for the excellent feedback.